src/test/service/delete.user.service.spec.ts

```diff
@@ -0,0 +1,83 @@
+import * as chai from "chai";
+import * as proxyquire from "proxyquire";
+import * as sinon from "sinon";
+import * as sinonChai from "sinon-chai";
+import * as nock from "nock";
+
+const expect = chai.expect;
+chai.use(sinonChai);
+
+describe("test delete user profile service", () => {
+
+  const deleteUserProfileURL = "http://localhost:4453/users";
+
+  let deleteUserProfile;
+
+  let req;
+
+  beforeEach(() => {
+    req = {
+      body: { idamId: "aaa@yahii.com" },
+      headers: {
+        Authorization: "userAuthToken",
+        ServiceAuthorization: "serviceAuthToken",
+      },
+      session: { jurisdiction: "test" },
+    };
+
+    const config = {
+      get: sinon.stub(),
+    };
+    config.get.withArgs("adminWeb.userprofiles_url").returns(deleteUserProfileURL);
+
+    deleteUserProfile = proxyquire("../../main/service/delete.user.profile.service", {
+      config,
+    }).deleteUserProfile;
+  });
+
+  it("should return an HTTP 201 status and success message", (done) => {
```

Comment:
This isn't correct - HTTP `201` is used to indicate something has been created, but this is a deletion! It should be checking for a `204`